lua/lint/linters/eslint.lua

```diff
@@ -24,17 +26,21 @@ return require('lint.util').inject_cmd_exe({
   stream = 'stdout',
   ignore_exitcode = true,
   parser = function(output)
-    local success, decodedData = pcall(vim.json.decode, output)
-    local diagnostics = {}
+    local success, decodedData = pcall(
+      vim.json.decode, output,
+      { luanil = { object = true, array = true } }
+    )
+    local messages = decodedData and decodedData[1] and decodedData[1].messages or {}
 
-    if success and decodedData ~= nil then
-      for _, diagnostic in ipairs(decodedData.messages or {}) do
+    local diagnostics = {}
```

Comment:
I wonder about the severities. In my local config I had to update `severties` because they did not match the actual ones:

```
    "off" or 0 - turn the rule off
    "warn" or 1 - turn the rule on as a warning (doesn’t affect exit code)
    "error" or 2 - turn the rule on as an error (exit code is 1 when triggered)
```
from https://eslint.org/docs/latest/use/configure/rules

Reply:
👍🏼 

This should now be fixed.